src/crud.js

```diff
@@ -0,0 +1,53 @@
+function editTaskEventListener(parentCheckBoxContainer) {
+  parentCheckBoxContainer.addEventListener('input', (event) => {
+    const { target } = event;
+    if (target.classList.value.indexOf('editable') !== -1) {
+      const tasksArray = JSON.parse(localStorage.getItem('tasksList'));
+      const inputWithId = target.previousElementSibling;
+      // eslint-disable-next-line max-len
+      const index = tasksArray.findIndex((element) => element.index === parseInt(inputWithId.id, 10));
+      tasksArray[index].description = target.innerHTML;
+      localStorage.setItem('tasksList', JSON.stringify(tasksArray));
+    }
+  });
+}
+
+const representNewIndexes = (tasksArray) => {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < tasksArray.length; i++) {
+    tasksArray[i].index = i;
+  }
+};
+
+const addNewTask = (newTask, incomingTasks) => {
+  let newTaskArray;
+  if (incomingTasks) {
+    newTaskArray = [...incomingTasks, newTask];
+  } else {
+    newTaskArray.push(newTask);
+  }
+  representNewIndexes(newTaskArray);
+  return newTaskArray;
+};
+
+const deleteTask = (index, tasksArray) => {
+  const filteredArray = tasksArray.filter((element) => element.index !== index);
+  return filteredArray;
+};
+
+const clearAllCompletedTasks = (completedButtonElement) => {
+  completedButtonElement.addEventListener('click', (event) => {
+    if (event.target.id === 'clear-button') {
+      const tasksArray = JSON.parse(localStorage.getItem('tasksList'));
+      const filteredArray = tasksArray.filter((element) => element.completed === false);
+      representNewIndexes(filteredArray);
+      localStorage.setItem('tasksList', JSON.stringify(filteredArray));
+      // eslint-disable-next-line no-restricted-globals
+      location.reload();
+    }
+  });
+};
+
```

Comment:
- [ ] Adding, deleting and editing a task function doesn't work, please work on it

![rr20](https://user-images.githubusercontent.com/55635977/134773516-ad1d00e8-e3e6-450d-8204-eb9f8dad9d3c.png)

- [ ] Your webpack.config.js file seem to have an issue, kindly check the configuration again to fix that issue.

![rr19](https://user-images.githubusercontent.com/55635977/134773514-140bb5ea-c650-43ba-80c8-c2f9a73cd328.png)